admin/server/auth.go

```diff
@@ -169,8 +169,11 @@ func (s *Server) user(c echo.Context) error {
 	}
 
 	var profiles map[string]interface{}
-	profile := sess.Values["profile"].([]byte)
-	json.Unmarshal(profile, &profiles)
+	profile, _ := sess.Values["profile"].([]byte)
+	err = json.Unmarshal(profile, &profiles)
+	if err != nil {
+		return c.String(http.StatusInternalServerError, err.Error())
+	}
```

Comment:
Actually, this is a safety regression. Previously, if it was not a `[]byte`, the type assertion would panic. Now it will fail silently and return an unmarshal error.

I feel ambivalent about this lint error. Let's disable it for type assertions for now (and restore this to what it was before).